ecl/hqlcpp/hqlhtcpp.cpp

```diff
@@ -16904,7 +16904,7 @@ ABoundActivity * HqlCppTranslator::doBuildActivityCreateRow(BuildCtx & ctx, IHql
     IHqlExpression * self = selfCursor->querySelector();
 
     if (isDataset)
-        associateSkipReturnMarker(funcctx, queryBoolExpr(false), selfCursor);
+        associateSkipReturnMarker(funcctx, queryZero(), selfCursor);
```

Comment:
This was previously wrong (but would have worked).  The return type of the function is integer, not boolean.